launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -45,8 +49,19 @@ def to_parameters_list(
     for params_set_or_path in evaluated_parameters:
         if isinstance(params_set_or_path, pathlib.Path):
             with open(str(params_set_or_path), 'r') as f:
+                if namespace is not None:
+                    node_name = f"{namespace}/{node_name}"
+                node_name_with_slash = f"/{node_name}"
```

Comment:
this might be duplicating the slash, right?
maybe:

```suggestion
                if not node_name.startswith('/'):
                  node_name_with_slash = f"/{node_name}"
```

Reply:
and you would also need to compute `node_name_without_slash` if that's the case, to always test for both names in the parameteter dict

Reply:
Good point. I think it might be easier to strip the slash here and when getting the name from the dict so we don't have to worry about comparing with & without the slash. Done in 97ede21